h2/src/main/org/h2/command/dml/Optimizer.java

```diff
@@ -234,15 +241,24 @@ private boolean shuffleTwo(TableFilter[] f) {
 
     /**
      * Calculate the best query plan to use.
+     *
+     * @param parse If we do not need to really get the best plan because it is view a parsing stage.
```

Comment:
I'm not sure what is the best here, the long description is an attempt to make purpose of the parameter clear.